package-lock.json

```diff
@@ -1,6 +0,0 @@
-{
-  "name": "amazon-chime-sdk",
-  "lockfileVersion": 2,
-  "requires": true,
-  "packages": {}
-}
```

Comment:
why is there a package-lock on the root level of this repo? We can delete it like this right?